kANe/v5p2/10_Hindu_Culture_and_Civilization/36_Characteristics_of_Hindu_Culture.md

```diff
@@ -102,93 +103,58 @@ The Mahābhārata passages quoted in n. 18 p. 8 of H. of Dh. Vol. II say 'the th
 
-A. M, T, Jackson says in 'Indian Antiquary' for 1910 p. 77' tho attractive power of Hindu civilization whicb has enabled it to assimilate and absorb into itself every foreign invader except the Moslem and Europeaa..,ladia civilized the nomads of Central Asia, so that wild Turkman tribes have been transformed into some of the most famous Rajput royal races,'
+[^2619]: It is remarkable that the late Lokamānya Tilak advocated about four months before his death (in the issue of the 'Kesari' of 16th March 1920) that anyone who spent his life in Indian freedom struggle must be treated as a brāhmaṇa to whatever caste or sub-caste-he might have belonged by birth, that caste should not be treated as a matter of birth but that we must begin to treat it as dependent on qualities and actions and he relies on our śāstras and quotes in support a Pali verse from the Suttanipāta : 'न जच्चा बसलो होति न जच्चा होति ब्राह्मणो । कम्मणा बसलो होति कम्मणा होति बाह्मणो.''a man does not become a Vṛṣala, that is, Śūdra by caste (or birth) nor does be become a brāhmaṇa by birth; a man becomes a vṛṣala or a brāhmaṇa by his actions' (compare Mahāvagga, vāseṭthasutta, verse 57 in SBE Vol. X part 2 p. 115 ). There is great misunderstanding about his opposition to those reformers who did not care for or engage in political agitation but spent their energies and time in inveighing against some social evils. He wanted to remove the evil of grinding poverty and political bondage of India due to the colonial policy of Britain for over 150 years and therefore be stressed the point that everyone must join in the fight for freedom and did not like the idea of some very intellectual men spending time and energy in crying themselves hoarse for only social reform. He thought that when India became free from the incubus of foreign rule, many social reforms would be introduced in no time, Events have shown that he was right, and the social reformers who maligned him were wrong. Our Constitution abolished untouchability at one stroke (Article 17 ), legislation has made a provision that to treat a man as untouchable in public is punishable (vide Act XX11 of 1955 sections 3 and 4 ), legislation has removed all ban against intercaste marriages and several other reforms have been brought about. Vide 'Social Legislation and its role in Social Welfare' published in 1956 on behalf of the Planning Commission. For Tilak's article, vide 'लोकमान्य टिळकांचे केसरींतील लेख भाग ३, पृष्ठे ४५३-५८1. British people were cautious and were generally opposed to progressive social reforms in Hindu Society. The High Courts administered Hindu Law as contained in the medieval digests (nibandhas) like the Dāyabhāga, the Mitakṣarā and the Vyavahāramayūkha and not the law found in ancient smṛtis, which allowed anuloma marriages; vide H. of Dh, vol. II pp. 448-452 and "A Century of Social reform' by S. Natarajan, Asia Publishing House, Bombay, particularly pp. 126–173.
```

Comment:
`'लोकमान्य टिळकांचे केसरींतील लेख भाग ३, पृष्ठे ४५३-५८1` → `'लोकमान्य टिळकांचे केसरींतील लेख भाग ३, पृष्ठे ४५३-५८1'`
 "A Century of Social reform' →"A Century of Social reform" (quotes must match.)

Reply:
corrected.